### DISCUSSION TITLE
2.0 version of october cms ?

### DISCUSSION THREAD

**Discussion by PubliAlex** [skydiver/wn-forms-plugin#3] [Q&A] [answered]
Hello,

I see that the 2.0 had some improvements (exemple : file uploading using filepond). I wonder if you plan to port that 2.0 version to october cms

Best regards,

**Comment by skydiver** [comment, marked as answer]
Hey @PubliAlex 

No plans to support OctoberCMS 😞

**Reply by PubliAlex** [reply]
Sad to hear, would you be ok with a fork of your module that support october cms ?

**Reply by skydiver** [reply]
sure, feel free to use as you wish, it's open source, not like October 2 😉

**Reply by PubliAlex** [reply]
October 2 is still open source, but need a paid license in order to be used 😃

**Reply by skydiver** [reply]
_Richard Stallman doesn't like that_  🤣

**Comment by WebVPF** [comment]
)))))))))))))))
